src/main/bash/sdkman-help.sh

```diff
@@ -27,6 +27,7 @@ function __sdk_help {
 	__sdkman_echo_no_colour "       list      or ls   [candidate]"
 	__sdkman_echo_no_colour "       use       or u    <candidate> <version>"
 	__sdkman_echo_no_colour "       default   or d    <candidate> [version]"
+	__sdkman_echo_no_colour "       home      or ho   <candidate> <version>"
```

Comment:
Same as before, make this `h` and drop `help` mnemonic.

Reply:
done